__tests__/app.test.js

```diff
@@ -91,3 +91,53 @@ describe('GET /api/articles', () => {
     });
 })
 
+describe('GET /api/articles/:article_id/comments', () => {
+    test('responds with status 200 and sends comments with correct keys and values', () => {
+        return request(app).get('/api/articles/9/comments').expect(200).then((res) => {
+            const comments = res.body.comments;
+            const expectComment = {
+                body: "Oh, I've got compassion running out of my nose, pal! I'm the Sultan of Sentiment!",
+                votes: 16,
+                author: "butter_bridge",
+                article_id: 9,
+                created_at: "2020-04-06T12:17:00.000Z",
+              }
+            
+            expect(comments).toEqual(expect.arrayContaining([expect.objectContaining(expectComment)]))
+        }) 
+    });
+
+    test('does not send comments from other articles, sends all comments for requested article', () => {
+        return request(app).get('/api/articles/1/comments').expect(200).then((res) => {
+            const comments = res.body.comments;
+            expect(comments).toHaveLength(11);
+            comments.forEach((comment) => {
+                expect(comment.article_id).toBe(1)
```

Comment:
We could be more thorough here and assert the shape of the comment: 
 - assert the object has the keys we expect 
 - assert that the typeof the property values

Reply:
Thanks for reviewing. I thought my first test covered the keys and values, is it not an appropriate test for this purpose? As it is a bit different to how I tested for this previously.